NEWS.md

```diff
@@ -79,6 +79,12 @@ function calls. (#850, #851, @renkun-ken)
 * Debugging functions (`browser()`, `debug()`, `debugcall()`, `debugonce()`, `trace()`, `undebug()`, `untrace()`) are now part of the default set of undesirable functions to help prevent them from being committed by mistake. (#876, @michaelchirico)
 * New linter `package_hooks_linter()` runs a series of checks also done by `R CMD check` on the `.onLoad()`, `.onAttach()`, `.Last.lib()` and `.onDetach()` hooks (#882, @MichaelChirico)
 * Improved location information for R parse errors (#894, #892, @renkun-ken and @AshesITR)
+* New tag based documentation pages for linters (#888, @AshesITR)
+  * Each linter has its own help page
+  * `?linters` also links to tag help pages, collecting linters with a similar goal
```

Comment:
idea (you may have mentioned this/similar before): it may be nice to have an API to set linters by tag in .lintr, WDYT?

Reply:
An idea might be to export alll those linter lists like `default_linters` and think of a smart way of deduplication?

e.g. make something like `linters: with_defaults(best_practices_linters, readability_linters)` work, returning all linters that are tagged any of default, best_practices or readability work?